corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html

```diff
@@ -40,10 +46,104 @@ <h1>{% trans "Searching the Case Lists" %}</h1>
     </div>
   </div>
 
-  <table id="report-datatable" class="table"></table>
+  <div id="report-datagrid">
+    <table class="datagrid-headers">
```

Comment:
out of curiosity, why are these different tables? seems like that requires that you specify the width of each column, or have some javascript to update the width if the text overflows. Not that we are particularly good at this elsewhere, but I think this might also be confusing from an accessibility standpoint.

Reply:
because the body table will be infinitely scrollable and will have a fixed height, and that should scroll independently from the headers. the `<tbody>` element itself is also not reliable in how scrolling is handled cross-browser wise, which it's why it's better to wrap it as a whole table independently. You can see how datatable handles this...it's the same method.